reana_job_controller/schemas.py

```diff
@@ -41,3 +41,5 @@ class JobRequest(Schema):
     voms_proxy = fields.Bool(required=False)
     kubernetes_uid = fields.Int(required=False)
     unpacked_img = fields.Bool(required=False)
+    htcondor_max_runtime = fields.Str(required=False)
```

Comment:
Nice, I didn't know we had marshmallow schemas in REANA. I wish we had these in `r-sever` and `r-w-c`...

Reply:
We should add these musings to the existing RFC https://github.com/reanahub/reana-job-controller/issues/106 if we want it to change.